.github/workflows/notebook-test-suite.yml

```diff
@@ -133,7 +134,11 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Pull Docker image
-        run: docker pull ghcr.io/nasa/regression-tests-${{ matrix.service }}:${{ github.event.client_payload.docker-image-tag || inputs.docker-image-tag || 'latest' }}
+        run: docker pull ghcr.io/${{github.repository_owner}}/regression-tests-${{ matrix.service }}:${{ github.event.client_payload.docker-image-tag || inputs.docker-image-tag || 'latest' }}
+
+      - name: Update Test Image Name
+        run: |
+          echo "REGRESSION_TESTS_$(echo "${{ matrix.service }}" | tr '[:lower:]' '[:upper:]' | tr '-' '_')_IMAGE=ghcr.io/${{ github.repository_owner }}/regression-tests-${{ matrix.service }}:${{ github.event.client_payload.docker-image-tag || inputs.docker-image-tag || 'latest' }}" >> $GITHUB_ENV
```

Comment:
This is a change I added to get a forked repo to be able to run the tests.  It's awkward, but I don't see other simple options.

Reply:
Not fully correct.  Makefile has nothing to do with the .github actions.
This is updating to remove the hard coded nasa from the docker pull via ${{github.repository_owner}} on L137

Then L141 is setting the REGRESSION_TESTS_[testname fixedup]_IMAGE to the value of the docker image that is desired/just built.

When the environment variable is set it is used in preference to the default image. Which is how you can choose which image should be used as opposed to the default.  now we're just setting this for each run, to the default value on the nasa github or your correct value in your fork.